openapi-schema.yml

```diff
@@ -1,8 +1,7 @@
 openapi: 3.0.2
```

Comment:
How did you generate this schema? Just curious as it seems to add some stuff that was missing. I have always just set this file to the output of `http://127.0.0.1:8000/api/schema?format=openapi` from a local instance but apparently that is missing some stuff...

Reply:
From the official Django Docs on RestFramework: https://www.django-rest-framework.org/api-guide/schemas/#generating-a-static-schema-with-the-generateschema-management-command

`python manage.py generateschema --file openapi-schema.yml`

Reply:
Cool. It’s interesting that the generated one from the API appears to be missing stuff. I’ll need to add a gulp command for that 👍